lib/bank_accounts/savings_account.rb

```diff
@@ -0,0 +1,81 @@
+module BankAccounts
+  class SavingsAccount < Account
+    ##--------------------------------------------------------------------------
+    # CONSTANTS
+
+    # limits
+    MINIMUM_BALANCE = 10
+
+    # fees
+    WITHDRAWAL_FEE = 2.00
+
+
+    ##--------------------------------------------------------------------------
+    # INSTANCE METHODS
+
+
+    # create a new savings account.
+    def initialize(id, initial_balance)
+      if initial_balance < MINIMUM_BALANCE
+        raise ArgumentError.new("You cannot create a savings account with less than a $#{ MINIMUM_BALANCE } initial deposit.")
+      end
+
+      super(id, initial_balance)
+
+      return true
```

Comment:
You shouldn't need any return here. I'm fairly noobish at Ruby (hello from cohort[1]!) but I've never seen an `initialize` method return anything.